web/src/js/ducks/ui/flow.js

```diff
@@ -1,18 +1,22 @@
 import * as flowsActions from '../flows'
+import { getDiff } from "../../utils"
+
 import _ from 'lodash'
 
 export const SET_CONTENT_VIEW = 'UI_FLOWVIEW_SET_CONTENT_VIEW',
              DISPLAY_LARGE    = 'UI_FLOWVIEW_DISPLAY_LARGE',
              SET_TAB          = "UI_FLOWVIEW_SET_TAB",
              START_EDIT       = 'UI_FLOWVIEW_START_EDIT',
              UPDATE_EDIT      = 'UI_FLOWVIEW_UPDATE_EDIT',
-             STOP_EDIT        = 'UI_FLOWVIEW_STOP_EDIT'
+             STOP_EDIT        = 'UI_FLOWVIEW_STOP_EDIT',
+             UPLOAD_CONTENT   = 'UI_FLOWVIEW_UPLOAD_CONTENT'
 
 
 const defaultState = {
     displayLarge: false,
     modifiedFlow: false,
     contentView: 'ViewAuto',
+    lastFileUpload: false,
```

Comment:
my problem here was that the contentloader compontent did not rerender because the flow does not really change because the content will not be updated.